crates/gcp/tests/context.rs

```diff
@@ -76,10 +76,12 @@ impl StorageIntegration for GcpIntegration {
         let account_path = self.temp_dir.path().join("gcs.json");
         info!("account_path: {account_path:?}");
         std::fs::write(&account_path, serde_json::to_vec(&token).unwrap()).unwrap();
-        std::env::set_var(
-            "GOOGLE_SERVICE_ACCOUNT",
-            account_path.as_path().to_str().unwrap(),
-        );
+        unsafe {
```

Comment:
Am I missing something, or did we remove `unsafe` around `set_var` further above, specifically in `crates/aws/src/storage.rs`, and here we are adding it?

Reply:
I have to add it back to the other one, thanks for pointing it out